pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -349,8 +357,7 @@ func (vm *windows) initializeBootstrapperFiles() error {
 	// Download the worker ignition to C:\Windows\Temp\ using the script that ignores the server cert
 	ignitionFileDownloadCmd := wgetIgnoreCertCmd + " -server " + vm.workerIgnitionEndpoint + " -output " +
 		winTemp + "worker.ign" + " -acceptHeader " + ignitionAcceptHeaderSpec
-	out, err := vm.Run(ignitionFileDownloadCmd, true)
-	log.V(1).Info("ignition file download", "cmd", ignitionFileDownloadCmd, "output", out)
```

Comment:
wouldn't having the out be helpful or we could log the error for vm.Run as well

Reply:
We are always [logging](https://github.com/openshift/windows-machine-config-operator/pull/147/files/9e894157568ff4faa08c5b95bf8291c38f4ec21b#diff-99dde3a6fcbd3bbbd7531252919d742735241faed60cf190a22c6c485ef92889R166) errors in `vm.Run`. Not sure if we need to do that here again.